docs/transport/transport.md

```diff
@@ -0,0 +1,65 @@
+# gittuf Remote Transfer Protocol
+
+Last Modified: July 1, 2024
+
+## Introduction
+
+Alongside the `gittuf` binary, gittuf ships with a custom remote transfer
+protocol binary. We call this the **transport** binary, named
+`git-remote-gittuf`.
+
+It's an easy way to get started with using gittuf on your repository, as it
+takes care of the following common operations for you:
+
+- Creating RSL entries upon pushing your changes
+- Verifying changes you pull down
+
+As time goes on, the transport will perform other "best-practice" operations
```

Comment:
I'd drop this. Let's just have a list of things it currently does, we can update as new features are added.